### ISSUE TITLE
[$30][MSFT-219] The NVDA doesn't read some items when user try to mouse hover on them

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2878]
**Steps to Reproduce**
1. Navigate to https://accounts.topcoder.com/sso-login/?app=member
2. Try to mouse hover on "email or handle" text-box or "Back to Login / Privacy Policy" links or "© 2019 TOPCODER. ALL RIGHTS RESERVED" text

**Expected Result**
The NVDA should read all items when user try to mouse hover on them

**Actual Result**
The NVDA doesn't read the "email or handle" text-box or "Back to Login / Privacy Policy" links or "© 2019 TOPCODER. ALL RIGHTS RESERVED" textwhen user try to mouse hover on them

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.2.3 Audio Description or Media Alternative (Prerecorded)
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F24

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096247 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096247 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096247 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by asadath1395** [comment]
@veshu This is not reproducible in latest NVDA version. Please verify

**Comment by mishacucicea** [comment]
@asadath1395 This talks about mouse-over, not switching through items with tab.
You can read more about it here: https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Metadata_On_Hover

**Comment by mishacucicea** [comment]
@drasticdpk Do you know any site where hovering on links/buttons work?

**Comment by sadathanwar17** [comment]
@mishacucicea I tried for mouse-over itself and it worked for me

**Comment by mishacucicea** [comment]
@tosha5252 / @drasticdpk Can you take a look over this issue to double-check?

**Comment by drasticdpk** [comment]
Hi @mishacucicea / @tosha5252, It worked me too (on NVDA version 2019.1.1). For now please close this issue.

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096771 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
